addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -243,6 +243,21 @@ function dragNDrop(snippet, position = "bottom") {
     };
 }
 
+function waitScrollToSnippet(snippet) {
+    return {
+        content: `Wait until scroll to ${snippet.name} completed`,
+        trigger: "iframe body:not(.o_scroll_to_snippet)",
+        isCheck: true,
+    };
+}
```

Comment:
Quickly scrolling through: that part would need huge explanation to explain why it is needed. IMO it probably is not: any operation done *during* a scroll should be effective. I think this would hide real bugs in tests if this is done automatically.

Reply:
Here is a summary of the related investigation:

In <https://runbot.odoo.com/runbot/batch/1359594/build/58886559>, the `scrollTo` was kept synchronous but the scroll delay was reduced to 1ms.
This led to the same test failures as when making the scroll asynchronous.
Conclusion: those tests accidentally worked thanks to the time spent waiting for `scrollTo`, but they should actually have been waiting something else

In <https://runbot.odoo.com/runbot/batch/1359883/build/58892590>, `s_popup`'s options and tests were adapted in order to wait for BS shown and hidden to be handled.
They do not fail anymore.

=> Indeed waiting for scroll was wrong to begin with.